packages/terra-tabs/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+* Added
+  * Added Common tabs component under terra-tabs
+  * Introduced `variant` prop to switch between framework and workspace styles
+
+* Changed
+  * Updated framework tabs to consume common tabs 
+  * Updated styles to accommodate both framework and workspace styles
+
+* Breaking Change
+  * Removed `responsiveTo` prop as common tabs is responsive to parent
+   container & window
```

Comment:
+1 on documenting the breaking change. 

Is there a release strategy for the the major version bump?